bot/code_coverage_bot/hooks/base.py

```diff
@@ -26,11 +26,25 @@
 
 class Hook(object):
     def __init__(
-        self, repository, revision, task_name_filter, cache_root, required_platforms=[]
+        self,
+        repository,
+        revision,
+        task_name_filter,
+        cache_root,
+        working_dir=None,
```

Comment:
What if we make working_dir a required arg (so we can avoid the if condition)?

Reply:
Done